torch/_tensor.py

```diff
@@ -1690,9 +1690,14 @@ def __dlpack__(self, stream=None):
             both streams.  If None or -1 is passed then no synchronization is performed.
             If 1 (on CUDA) or 0 (on ROCM) then the default stream is used for
             synchronization.
+
+            max_version (tuple[int, int] or None): An optional Python tuple with
```

Comment:
whats the tradeoff with being conservative on the default value?

Reply:
That's correct.